Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx

```diff
@@ -50,9 +74,34 @@ export const UnbilledExpensesContainer = () => {
             Enter New <FontAwesomeIcon icon={faPlus} />
           </Link>
         </div>
+
+        <Modal isOpen={selectedExpense !== null}>
+          <ModalHeader>Modal title</ModalHeader>
+          <ModalBody>
+            Are you sure you want to remove this unbilled expense?
+          </ModalBody>
+          <ModalFooter>
+            <Button
+              color="success"
+              onClick={() =>
+                selectedExpense !== null
+                  ? confirmModal(selectedExpense.id)
+                  : null
+              }
+            >
+              Confirm
+            </Button>{" "}
```

Comment:
What is the {" "} for?

Reply:
It creates space between the two buttons

Reply:
Hmm, I removed it and didn't notice any difference.
Probably better to use a class for this kind of formatting. @cydoval ?

Reply:
Empty string is actually pretty common way to do things. But yea there might be something better. But you do see it this way a lot, especially when prettier reformats things.